contracts/security/ReentrancyGuard.sol

```diff
@@ -55,7 +55,7 @@ abstract contract ReentrancyGuard {
 
     function _nonReentrantBefore() private {
         // On the first call to nonReentrant, _status will be _NOT_ENTERED
-        require(_status != _ENTERED, "ReentrancyGuard: reentrant call");
+        require(_status == _NOT_ENTERED, "ReentrancyGuard: reentrant call");
```

Comment:
This causes an issue if the module is added through an upgrade. If for any reason the constructor/initialized is not set, `_status` will be 0, and the system will be blocked